sb/robot/game_specific.py

```diff
@@ -0,0 +1,47 @@
+WALL = set(range(0, 28))  # 0 - 27
```

Comment:
Almost certainly an issue for a separate PR, but IMO it feels cleaner to import these constants from `robot-api` (which is presumably where they've been copied from) rather than maintaining two copies of this list.

Reply:
(this is an issue with the API however so I don't want to fix it here)